src/CalculatorUITestFramework/CalcHistoryFlyout.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+using System.Collections.ObjectModel;
+using OpenQA.Selenium.Interactions;
+using System.Windows.Input;
+
+namespace CalculatorUITestFramework
+{
+    public class CalcHistoryFlyout
+    {
+        private WindowsDriver<WindowsElement> session => WinAppDriver.Instance.CalculatorSession;
+        public HistoryPanel HistoryPanel = new HistoryPanel();
+        public WindowsElement Header => this.session.TryFindElementByAccessibilityId("Header");
+        public WindowsElement AppName => this.session.TryFindElementByAccessibilityId("AppName");
```

Comment:
I think it's a little odd to have the Header and Appname properties here.  From test case code, I would be a bit confused if I saw "myHistoryPanel.AppName" giving me back an element.  Perhaps we should remove these and just use the session directly?  More generally speaking, it's a good idea to keep things that you don't expect people to consume from test code private.

Reply:
These are used to send keys in the test.  Header is used throughout most of standard tests.  However, since calculator doesn't have a header in when it is in KOT(keep on top) mode, than AppName is used.

Reply:
I get why you have those, but I don't think this is the right place for them.  From a test author's perspective, the AppName element exists on the whole calculator and not the HistoryPanel.  I don't think we have a central place to put appwide elements, but perhaps we should create one?  Alternatively, making it private instead of public can help too as it becomes more of an implementation detail.

Reply:
I think it's a good idea to have a global calculator cs that contains global calculator elements like these.  I will add that.